ldk/csharp/OliveHelpsLDK/SelfTestLoop/TestAttribute.cs

```diff
@@ -0,0 +1,13 @@
+namespace SelfTestLoop
+{
+    [System.AttributeUsage(System.AttributeTargets.Method)]
+    public class TestAttribute : System.Attribute
+    {
+        public string Name { get; set; }
+
+        public TestAttribute(string name)
+        {
+            Name = name;
```

Comment:
I'm a bit confused why do we need to expose TestAttribute(string name) here. "Name" is already public and has public getter and setter. Perhaps it is extra flexibility.